src/helperFunctions/hash.py

```diff
@@ -45,5 +52,19 @@ def check_similarity_of_sets(pair_of_sets, all_sets):
     return True
 
 
+def _is_elf_file(file_object):
+    file_type = file_object.processed_analysis['file_type']['full'].lower()
+    return re.search(r'elf', file_type) is not None
```

Comment:
this will produce many false positives
you might check for 'application/x-executable', 'application/x-object' and 'application/x-sharedlib' in ['file_type']['mime']
or use another regex like r'elf \d\d-bit'